swap_meet/clothing.py

```diff
@@ -1,2 +1,15 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+
+class Clothing(Item):
+
+    
+    def __init__(self, condition = 0, age = None):
+        self.category = "Clothing"
+        self.condition = condition
+        self.age = age
```

Comment:
This will work, but it's duplicating code that is already in Item.  Using the super constructor allows this function to take advantage of all of the code in Item:
```suggestion
        super().__init__("Clothing", condition, age)
```